seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/TimeFormat.java

```diff
@@ -29,4 +35,19 @@ public class TimeFormat {
                     .appendPattern("HH:mm:ss")
                     .appendFraction(ChronoField.NANO_OF_SECOND, 0, MAX_TIME_PRECISION, true)
                     .toFormatter();
+    public static final DateTimeFormatter CUSTOM_DATE_TIME_FORMAT =
+            new DateTimeFormatterBuilder()
+                    .parseCaseInsensitive()
+                    .append(ISO_LOCAL_DATE)
+                    .appendLiteral(' ')
+                    .append(ISO_LOCAL_TIME)
+                    .toFormatter();
+
+    public static TemporalAccessor parseDateTime(CharSequence text) {
```

Comment:
this try-catch in sometime will cause time format inconsistent.
like some data can be parsed with ISO_LOCAL_DATE_TIME, others can't.